toolchain/check/pattern.h

```diff
@@ -42,6 +42,11 @@ auto AddBindingPattern(Context& context, SemIR::LocId name_loc,
                        SemIR::ExprRegionId type_region_id, bool is_generic,
                        bool is_template) -> BindingPatternInfo;
 
+// Creates storage for `var` patterns nested within the given pattern at the
+// current location in the output SemIR.
+auto AddPatternVarStorage(Context& context, SemIR::InstBlockId pattern_block_id,
+                          bool returned) -> void;
```

Comment:
This took me a bit to unwrap, suggesting a slightly different name, but separately/also the comment note above (just looking for a note about the semantic of this).
```suggestion
                          bool is_returned_var) -> void;
```

Reply:
Not sure if this comment was missed. FWIW, my suggesting was really from interpreting "returned" as relating to "is the result returned" by something like `var x ...; return x;` rather than `returned var`.

Reply:
Done.